main.go

```diff
@@ -156,6 +157,11 @@ func main() {
 			`,
 				strings.Join(packagesList, "\n"),
 				code)
+			formatted, err := format.Source([]byte(code))
+			if err != nil {
+				log.Fatal(err)
+			}
+			code = fmt.Sprintf("%s", formatted)
```

Comment:
This is the source of one of the CI errors I think

Reply:
Fixed in bf6ea45